composer.json

```diff
@@ -40,13 +40,13 @@
     },
     "require" : {
         "php" : ">=5.4.0",
-        "squizlabs/php_codesniffer": "^3.5",
-        "phpcsstandards/phpcsutils": "^1.0"
+        "squizlabs/php_codesniffer": "^3.5"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.0 || ^6.5 || ^7.0 || ^8.0",
         "sirbrillig/phpcs-import-detection": "^1.1",
         "limedeck/phpunit-detailed-printer": "^3.1 || ^4.0 || ^5.0",
-        "phpstan/phpstan": "^0.11.8"
+        "phpstan/phpstan": "^0.11.8",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0"
```

Comment:
Please widen the version constraints for this to prevent conflicts with other standards requiring the plugin and/or project using this standard which may themselves require the plugin.

Reply:
Can do. Does that make a big difference, though, since this is a `require-dev` section? Just trying to learn.

Reply:
Never mind. I'd overlooked that this was `require-dev`. :+1: